### DISCUSSION TITLE
How to calculate the pressure in a sample case

### DISCUSSION THREAD

**Discussion by kouki0000** [DualSPHysics/DualSPHysics#251] [DUALSPHYSICS_V5.2]
In “03_Sloshing” and “04_Dambreak”, the pressure is calculated by “-onlytype:+all,+fluid” and “-onlytype:-all,+fluid”. What is the difference?

**Comment by alexbexe** [comment]
“-onlytype:+all,+fluid” will use pressure of all particles arround the pressure gauge
 “-onlytype:-all,+fluid” will only use the pressure of the fluid particles around the pressure gauge
